.editorconfig

```diff
@@ -2,7 +2,6 @@ root = true
 
 [*]
 charset = utf-8
-end_of_line = lf
```

Comment:
Can I ask the reason of this change?

Reply:
Well, the issue is that the default behavior for git on Windows (and the behavior recommended by git) is to use CRLF line endings and have them automatically normalized. Requiring contributors' editors to insert LF line endings means that you'll end up with files with both kinds of line endings, which causes git to emit warnings. It seems unnecessary since `.gitattributes` should enforce line ending normalization.

Reply:
It also means that you can have a file that looks unedited, but that `git status` will report has been modified, if you accidentally replace one line ending with another.